compiler/luci/import/include/luci/Importer.h

```diff
@@ -36,7 +36,12 @@ class Importer final
   Importer();
 
 public:
-  explicit Importer(const GraphBuilderSource *source) : _source{source}
+  explicit Importer(const GraphBuilderSource *source) : _source(source)
```

Comment:
Do you have to change this?

Reply:
No, it's just a strange diff =)

I moved introduced ctor below of existing one (without any modifying) in this changes.